libc/docs/CMakeLists.txt

```diff
@@ -4,6 +4,33 @@ include(AddSphinxTarget)
 if (SPHINX_FOUND)
   if (${SPHINX_OUTPUT_HTML})
     add_sphinx_target(html libc)
+
+    list(APPEND docgen_list
+      assert
+      ctype
+      errno
+      fenv
+      float
+      inttypes
+      locale
+      setjmp
+      signal
+      stdbit
+      stdlib
+      string
+      strings
```

Comment:
I wonder why these are being highlighted? Are they keywords in CMake? Perhaps I should make this a `;` separated string?